src/App.jsx

```diff
@@ -1,104 +1,27 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+function getComentsById(postId) {
+  return commentsFromServer.sort(comment => comment.postId === postId) || null;
```

Comment:
```suggestion
  return commentsFromServer.filter(comment => comment.postId === postId) || null;
```
you need to filter the array

Reply:
Still not fixed